Create 16-11-22_writeup.md (#184)
very-good-science/data-ethics-club: site/write_ups/2022/16-11-22_writeup.md

How do we value aesthetics? Is some of it what people believe about their own art? There is definitely capitalism and social bias in art. If (rich) people want a piece then it's consider more valuable. 

Something not mentioned in the article but seems like a big issue is to do with value of the artist themselves. Like with restaurants - you'll go to a famous chef's restaurant but the chef won't cook your meal. It's their ideas that you are paying for. 

Should skill be compensated? If you train for years to hone skill to create a certain type of art. With AI tools, now you can just type in a sentence and get out an image with little to no extra input. Does the quality of art scale somehow with effort that went in to produce it or skill level required? Our gut instincts may say yes, but is that actually true?

Will there become a problem with "Seen one seen them all" where AI generated art becomes too recognisable? Perhaps there is a comparison to fan fiction; it's usually non-comercial, so no ethical problems here. Most fan fiction isn't good; should the law be different if the art isn't good? Can we have a cultural shift around how you value things? Around how things sell?

> At the end of the day, art is what you make of it! On a purely philosophical level, it has value if it speaks to you, that’s it! Value of art is culturally constructed, Somali women creating structures by weaving don’t view it as art, but non-Somali cultures may well see it as art! It's highly contextual.	Does the desire define the value? Desire can be derived from various places, may be due to aesthetic value, or the exclusivity, or the authorship. 

Reviewer: ```suggestion
At the end of the day, art is what you make of it! On a purely philosophical level, it has value if it speaks to you, that’s it! The value of art is socially and culturally constructed.
For example, Somali women who create structures by weaving may not view it as art, but in non-Somali cultures it may be more commonly seen as art. It's highly contextual.	
Does the desire define the value? Desire can be derived from various places, may be due to aesthetic value, or the exclusivity, or the authorship. 
```

## Can we use AI art tools to disrupt current and futures narratives about the value of art works?

> In inclusive futures, when it comes to art especially in museums there are cultures where art has been taken from somewhere else. With all this art that's being generated, is it going to include all these other cultures as well? What's the skew of the datasets? What's the bias of the datasets? If you take samples from tumblr how will that be different to samples from instagram for example. Also how are things labelled? Who labels them?

Reviewer: ```suggestion
In inclusive futures, when it comes to art especially in museums there are cultures where art has been taken from somewhere else. With all this art that's being generated, is it going to appropriate other cultures all over again? What's the skew of the datasets? What's the bias of the datasets? If you take samples from tumblr how will that be different to samples from instagram for example. Also how are things labelled? Who labels them?
```

> Keir has noted in his own research that AI generated pictures of black women appear to have more skin showing that same prompts of white women. Is this cultural or due to labelling? 

Reviewer: ```suggestion
Keir has noted in his own research that AI generated pictures of black women appear to have more skin showing than same prompts of white women. Is this do to training images, or labelling? 
```

The input images are impossible to curate! There are already discussions about how Midjourney creates images which reinforce orientalism.

Some areas of art will no longer be monetisable; potential for those areas of art to stagnate. If AI art floods online repositories from which AI art draws, will we just create an endless mirror chamber of AI art?

Is there any reason why we couldn't have Infinite Monets? We could generate art in his style using AI, and people appreciate them aesthetically, but and AI work lacks e.g. the same intentionality. Real physical art can actually change over time, is AI art as unique or thrilling? Inevitably will not mean as much for some people

We change alongside art! Would we feel the same way about how our perception of art changes over time if the art is AI generated?

## Closing Thoughts

> The automation here comes from previous and ongoing work, not just from say embodied physical labour. We are used to train AI in image identification and now the next step seems to be image generation.

Reviewer: ```suggestion
The automation here comes from previous and ongoing work, not just from say embodied physical labour. We are used to using AI for image identification and now the next step seems to be image generation.
```

Who is giving consent for this data to be used? Where is the right place to issue consent for the usage? Is issuing under creative commons a means of giving consent for this sort of data usage? Does AI require a different regulatory environment? [Artist Greg Rutowski thinks that living artists works shouldn't be included in inputs](https://news.artnet.com/art-world/a-i-should-exclude-living-artists-from-its-database-says-one-painter-whose-works-were-used-to-fuel-image-generators-2178352).

What does "use" mean for AI training data. Is "Training data" a misleading term here? The previous paradigm doesn’t work under the sheer scale and "industrialisation" of AI, the contract can change because the tools are changing!

Can we have a cultural shift around how you value things? Around how things sell? How would we do that? 

What does this mean for AI usage in public services? We want tools to be trained so they can be better, but can we meaningfully get consent from e.g. a child in care?

> 

Reviewer: ```suggestion

As usual, we came away from this session with a lot more questions than answers! However, it was an intriguing and very busy discussion which made us all think a lot about what it means for art to have value and what it means to involve AI in it. 
```
